src/main/plugins/org.dita.base/build_init.xml

```diff
@@ -85,6 +85,10 @@ See the accompanying LICENSE file for applicable license.
       </condition>
     </fail>
     <property name="output.dir" location="${basedir}/out" />
+    <condition property="ditaot.render.dir" value="${dita.temp.dir}${file.separator}${temp.render.dir.name}" else="${output.dir}">
+      <isset property="temp.render.dir.name"/>
+    </condition>
+    <property name="ditaot.render.dir" location="${output.dir}"/>
```

Comment:
This is not needed because the else-branch of the preceding `condition` will set the property value.